hvm/cmd/list-releases.go

```diff
@@ -0,0 +1,46 @@
+package cmd
+
+import (
+	"fmt"
+
+	"github.com/chanzuckerberg/happy/shared/githubconnector"
+	"github.com/spf13/cobra"
+)
+
+// useCmd represents the use command
+var listRelasesCommand = &cobra.Command{
+	Use: "list-releases [org] [project]",
+
+	Short: "Get list of available releases",
+	Long:  ``, Run: func(cmd *cobra.Command, args []string) {
+		fmt.Println("use called")
+	},
```

Comment:
debugging leftover?

Reply:
Woops. Yeah. The `use` command even went away. Will clean this up.